README.md

```diff
@@ -5,7 +5,7 @@
 install with npm:
 
 ```sh
-npm install astro-windicss -D
+pnpm install astro-windicss -D
```

Comment:
i think specifying package manager in documention was not much good advice, maybe add extra information